paper/mel_report.Rmd

```diff
@@ -31,37 +34,237 @@ knitr::opts_chunk$set(fig.align = 'center',
-| Outcome 7: Increase in clients identified who are victims of GBV                                                     | Indicator 7: % increase in GBV clients identified                                         | Two timepoints: pre-training and post-intervention* *This measure can be used to compare to health facility reporting (which is the official MEL data source) on clients identified | Percentage compared to baseline | 50% increase   |
+## Outcome 4: Health providers who have participated in the ‘Responding to GBV Learning Lab’ Curriculum demonstrate improved knowledge related to GBV.
+When assessing indicator 4, the percentage of health facilities demonstrating improvement in staff knowledge, attitude, and empathy in responding to GBV, the project exceeded the MEL target, with 90% of   facilities demonstrating improvement in staff knowledge, attitudes, and empathy in responding to GBV (table 3). Additional analysis of this indicator can be found in the Appendix.
 
 ```{r}
-table <- difference_by_facility %>%
+outcome4_table <- difference_by_facility %>%
   select(region, outcome4_mean_value_1, outcome4_mean_value_3, outcome4_difference) %>%
 flextable()
 
-table <- set_header_labels(
-  table,
-  region = "Region",
-  outcome4_mean_value_1 = "Mean score timepoint 1", 
-  outcome4_mean_value_3 = "Mean score timepoint 3", 
-  outcome4_difference = "Difference in mean scores"
-)
-table
+outcome4_table <- set_caption(outcome4_table, "**Table 3. Outcome 4: Mean Provider Knowledge Score by Facility Catchment Area**")
+
+outcome4_table <- set_header_labels(
+  outcome4_table,
+  region = "Catchment area",
+  outcome4_mean_value_1 = "Baseline",
+  outcome4_mean_value_3 = "Endline",
+  outcome4_difference = "Percent Difference")
+
+footer_values <- c("All Facilities", 
+                   mean(difference_by_facility$outcome4_mean_value_1, na.rm = TRUE),
+                   mean(difference_by_facility$outcome4_mean_value_3, na.rm = TRUE),
+                   mean(difference_by_facility$outcome4_difference, na.rm = TRUE))
+
+outcome4_table <- add_footer_row(outcome4_table, values = footer_values, colwidths = c(1,1,1,1))
+
+outcome4_table
 ```
 
+## Outcome 5: Health providers who have participated in the ‘Responding to GBV Learning Lab’ have increased confidence in delivering survivor-centered care.
+When comparing the results to indicator 5, which is the percentage of health facilities that demonstrate improvement in health provider confidence in responding to GBV, the project exceeded the MEL target, with 100% of facilities demonstrating improvement in staff confidence in responding to GBV (table 4). Additional analysis of this indicator can be found in the Appendix.
 ```{r}
```

Comment:
Edit: 
When evaluating Indicator 5, which gauges the percentage of health facilities showcasing enhanced health provider confidence in GBV response, the project not only met but exceeded the MEL (Monitoring, Evaluation, and Learning) target. Specifically, Table 4 reveals that all facilities, constituting 100%, demonstrated improvements in staff confidence when responding to GBV. Further analysis of this indicator is available in the Appendix.